wiki/Help_Centre/Account/fr.md

```diff
@@ -228,7 +223,7 @@ Cela ne signifie pas non plus que vous pouvez créer un nouveau compte pour fair
 
 *Attention : La suppression d'un compte est permanente, et ne peut donc pas être annulée !*
 
-Oui, tout utilisateur peut demander la suppression de son compte en envoyant un courriel à [privacy@ppy.sh](mailto:privacy@ppy.sh). Gardez à l'esprit que la suppression de votre compte est **définitive**, ce qui signifie qu'elle ne peut pas être annulée. La suppression de votre compte ne signifie pas que vous pouvez créer un nouveau compte, car le fait d'avoir plusieurs comptes est contraire aux [règles](/wiki/Rules#règles-de-communauté). Cela signifie que vous ne pourrez plus utiliser aucune des fonctionnalités en ligne d'osu !
+Oui, tout utilisateur peut demander la suppression de son compte en envoyant un courriel à [privacy@ppy.sh](mailto:privacy@ppy.sh). Gardez à l'esprit que la suppression de votre compte est **définitive**, ce qui signifie qu'elle ne peut pas être annulée. La suppression de votre compte ne signifie pas que vous pouvez créer un nouveau compte, car le fait d'avoir plusieurs comptes est contraire aux [règles](/wiki/Rules#règles-de-communauté). Cela signifie que vous ne pourrez plus utiliser aucune des fonctionnalités en ligne d'osu! !
```

Comment:
[ASC](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#osu!) forbids `osu!!` but I guess it might technically work in French as `osu! !` since it separates the exclamation points. Nevertheless, there are no other instances of this so I recommend you reword this anyway.